tests/torch/data/reference_graphs/quantized/hw/NPU/inception_v3.dot

```diff
@@ -328,212 +328,212 @@ strict digraph  {
 "326 Inception3/view_0" [id=326, type=view];
```

Comment:
The quantization configuration is aligned between CPU and NPU. Could you remove references for INT8 NPU?

Reply:
Why should we remove these tests at all?
Isn't it better to update them to the mixed configuration that was used to generate references?
@ljaljushkin, what do you think?

Reply:
@alexsu52, I've removed references for the **PyTorch** part of the **NPU** tests.
But for the **Tensorflow** part, it's not so easy because of the following:

1. TF references are represented as .pb files that contain much more information about the model;
2. **CPU** references for TF contain the results (7bit quantizers) of the **overflow_fix** enabled option that is not applicable for **NPU** at all;
3. Even some connections in the resulting graphs are different between **CPU** and **NPU**;

Also, based on the reasons above, there are questions related to the PyTorch tests:

1. Why the **overflow_fix** was not applied for this backend?
2. Which information should be tested in such tests, since PyTorch contains only .dot files (structure) and Tensorflow - .pb (with the detailed information)?

Reply:
After the discussion with @alexsu52, we've decided to keep tests in their current state and solve the new issues with **half-precision** and **unified scales** in the future.